docs/rules/assertion-message.md

```diff
@@ -0,0 +1,47 @@
+# Enforce or disallow assertion messages
+
+Assertion messages are optional arguments that can be given to any assertion call to improve the error message, should the assertion fail. This rule either enforces or disallows the use of those messages.
+
+
+## Fail
+
+```js
+import test from 'ava';
+
+/*eslint ava/assertion-message: ["error", "always"]*/
+test(t => {
+	t.truthy(array.indexOf(value) !== -1);
```

Comment:
@novemberborn I was thinking about the other day too. Open a new issue and we can discuss there.